DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionDebugMenu.swift

```diff
@@ -69,18 +69,23 @@ final class NetworkProtectionDebugMenu: NSMenu {
                 NSMenuItem(title: "Reset All State", action: #selector(NetworkProtectionDebugMenu.resetAllState))
                     .targetting(self)
 
-                NSMenuItem(title: "Reset Site Issue Alert", action: #selector(NetworkProtectionDebugMenu.resetSiteIssuesAlert(_:)))
+                NSMenuItem.separator() // Resetting single components should go below this point
+
+                NSMenuItem(title: "Remove Network Extension and Login Items", action: #selector(NetworkProtectionDebugMenu.removeSystemExtensionAndAgents))
+                    .targetting(self)
+
+                NSMenuItem(title: "Remove VPN configuration", action: #selector(NetworkProtectionDebugMenu.removeVPNConfiguration(_:)))
                     .targetting(self)
 
                 resetToDefaults
                     .targetting(self)
 
-                NSMenuItem.separator()
+                NSMenuItem.separator() // Resetting VPN subfeatures should go below this point
 
-                NSMenuItem(title: "Remove Network Extension and Login Items", action: #selector(NetworkProtectionDebugMenu.removeSystemExtensionAndAgents))
+                NSMenuItem(title: "Reset Enforce Routes force-enabled flag", action: #selector(NetworkProtectionDebugMenu.resetEnforceRoutesForceEnabledFlag(_:)))
                     .targetting(self)
 
-                NSMenuItem(title: "Remove VPN configuration", action: #selector(NetworkProtectionDebugMenu.removeVPNConfiguration(_:)))
+                NSMenuItem(title: "Reset Site Issue Alert", action: #selector(NetworkProtectionDebugMenu.resetSiteIssuesAlert(_:)))
```

Comment:
This was moved to organize the reset options: first go the general reset options, then single components, then subfeatures.